packages/devextreme/js/__internal/ui/m_multi_view.ts

```diff
@@ -110,6 +110,12 @@ const MultiView = CollectionWidget.inherit({
       index -= count;
     }
 
+    const step = this._swipeDirection > 0 ? -1 : 1;
```

Comment:
Kindly rebase your branch to exclude changes u've made in the previous PR

Reply:
[24dc0c3](https://github.com/DevExpress/DevExtreme/pull/28137/commits/24dc0c3e7e438483f1744fc328f350756b34c362)